SimpleXML/SimpleXML.php

```diff
@@ -24,7 +24,7 @@ class SimpleXMLElement implements Traversable {
 	/**
      * Provides access to element's children
      * @param $name child name
-     * @return SimpleXMLElement[]
+     * @return array($this)
```

Comment:
`static[]`, but actually isn't related at all, because PhpStorm will just mark `$xml->property` as not found in class.